src/components/FileRepo.js

```diff
@@ -145,11 +146,26 @@ const FileRepo = ({ state, effects, ...props }) => {
                 streamData={props.streamData(props.index, props.projectId)}
               />
             </div>
+            {state.modalState.isShown && (
+              <AdvancedFacetViewModal
+                sqon={selectionSQON}
```

Comment:
This should propbably be `props.sqon`. `selectionSQON` is the current rows of the table that are selected.